src/controllers/action-queue/display.ts

```diff
@@ -176,11 +175,6 @@ async function getActionOpDisplayInfo(account: EdgeAccount, actionOp: ActionOp,
         ...baseDisplayInfo
       }
     }
-    default: {
```

Comment:
Let's continue to throw for the default case.

I prefer the switch pattern where default cases are always required, it makes it easier to pinpoint root causes of bugs quicker in the future.

Reply:
Ah, so this whole default case was a Flow hack. TypeScript does exhaustive type-checking and it will not pass if a case is missed.